apps/experiments/filters.py

```diff
@@ -8,6 +8,34 @@
 from apps.chat.models import Chat, ChatMessage
 
 
+class Operators:
```

Comment:
Not actually an enum. It should probably extend `StrEnum`

Reply:
See https://github.com/dimagi/open-chat-studio/pull/1501